forms-flow-web/src/services/UserService.js

```diff
@@ -148,11 +148,13 @@ const getFormioToken = () => localStorage.getItem("formioToken");
 // const doLogin = KeycloakData.login;
 // const doLogout = KeycloakData.logout;
 const getToken = () => KeycloakData?.token;
+const getUserInfo = () => KeycloakData?.userInfo;
 
 const UserService = {
   initKeycloak,
   userLogout,
   getToken,
+  getUserInfo,
```

Comment:
Do you mind moving `getUserInfo ` to the last item (after `setKeycloakJson`)? It would make the upgrade and merge conflict easier for the next update.

Reply:
That make sense.